config/manager/manager.yaml

```diff
@@ -26,6 +26,7 @@ spec:
       - command:
         - /manager
         args:
+          - "--audit-from-cache=false"
```

Comment:
Do we need to set this flag? Can we just rely on the default value?

Reply:
Yes we can so this is redundant. I left it in for illustrative purposes but I can remove it.

Reply:
Better to remove it, the less duplicated config we have the better.